interface/patient_file/void_dialog.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/* Copyright (C) 2016-2021 Rod Roark <rod@sunsetsystems.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ */
+
+$fake_register_globals = false;
+$sanitize_all_escapes  = true;
```

Comment:
The above 2 horrific lines have been exiled from OpenEMR. :bomb: 
:)

Reply:
Come to think of it, they do have historical values. We could turn them into a OpenEMR legacy NFT and sell them to the highest bidder :)
https://opensea.io/accounts/OpenEMR